code/modules/shuttle/emergency.dm

```diff
@@ -83,6 +83,8 @@
 			. = authorize(user)
 
 		if("repeal")
+			if(!(ID in authorized))
```

Comment:
Prevents retracting auth without having given it.
Stops possible log/admin spam
[2020-12-16 09:16:26.080] GAME: bloons3/(Ashlie Beedell) has deauthorized early shuttle launch in (205,145,2) 0 of 3 needed.
[2020-12-16 09:16:27.596] GAME: bloons3/(Ashlie Beedell) has deauthorized early shuttle launch in (205,145,2) 0 of 3 needed.